### PR TITLE
Remove v1alpha1 crd

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- remove v1alpha1 crd
- remove v1alpha pkg

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/16974

### CODE REVIEW

File: components/function-controller/pkg/apis/serverless/v1alpha2/function_defaults_test.go [hunk 1]
```diff
@@ -401,21 +399,13 @@ func TestSetDefaults(t *testing.T) {
 		},
 		"Should ignore label replicas-preset": {
```
Comment (anoipm) [inline components/function-controller/pkg/apis/serverless/v1alpha2/function_defaults_test.go:400]: Maybe we should remove whole this test? IMO without `v1alpha1ReplicasPresetLabel` it has no sense.

**Review by anoipm — DISMISSED** [review]

**Review by janmedrek — DISMISSED** [review]

**Review by anoipm — APPROVED** [review]

**Review by anoipm — APPROVED** [review]

**Review by anoipm — DISMISSED** [review]

**Review by anoipm — APPROVED** [review]

**Pull request merged**